File: python/tools/latn_iast_transliteration_verification/deva_to_gujr_test.py
from __future__ import annotations

from lipimala.deva_gujr_converter import (
    to_canonical_gujarati_from_devanagari,
)
from tools.fixtures.example_deva import devanagari_smoke_samples


def main() -> None:
    print('----------------------------------------------------------------')
    print(' DEVANAGARI TO GUJARATI SCRIPT CONVERSION')
    print('----------------------------------------------------------------')

    for source in devanagari_smoke_samples:
        result = to_canonical_gujarati_from_devanagari(source)
        print(f'"{source}" -> "{result}"')


if __name__ == '__main__':
    main()

File: dart/tools/latn_iast_transliteration_verification/deva_to_gujr_test.dart
// ignore_for_file: avoid_print

import 'dart:convert';

import 'package:indic_script_converter/deva_gujr_converter.dart';
import 'example_deva.dart' as examples;

void main() {
  print('----------------------------------------------------------------');
  print(' DEVANAGARI TO GUJARATI SCRIPT CONVERSION');
  print('----------------------------------------------------------------');

  for (final source in examples.devanagariSmokeSamples) {
    final result = source.toCanonicalGujaratiFromDevanagari();
    print('"$source" -> "$result"');
  }
}
